xmppserver/src/main/java/org/jivesoftware/openfire/auth/AuthProviderMapper.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 IgniteRealtime.org
+ * Copyright (C) 2016 IgniteRealtime.org, 2017-2018 Ignite Realtime Foundation. All rights reserved
```

Comment:
This happens in various places. I'm not against merging 'IgniteRealtime.org' (which in itself isn't a legal entity anyway - but it is 'us' so I'm happy to apply a change) with 'Ignite Realtime Foundation'.

Reply:
I merged the ones that are 2017 onwards, but kept 2016. I used the Ignite Realtime Foundation announcement blog post as the deciding date, which is October 2016, so there will have been commits that assign 'IgniteRealtime.org' from before the Foundation was announced. Playing it safe.